.github/workflows/node.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: |
           flatpak --user remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
           flatpak --user install -y flathub \
-            org.freedesktop.{Platform,Sdk{,.Extension.node{14,16}}}//22.08
+            org.freedesktop.{Platform,Sdk{,.Extension.node{14,16,18}}}//22.08
```

Comment:
Does this actually include npm 9? I noticed the comments on the other issue + the release flow seem to imply it hadn't been backported yet...

Reply:
No, node18 still has npm 8. I thought about bundling npm 9 at build time, but concluded that the fact that node18 understands lockfiles v3 is enough, since node is only reading lockfiles during our tests, and generated the lockfiles with npm 9 installed on host.

Reply:
Ah, it seems that [npm 9 is also going to make it to to node 18](https://github.blog/changelog/2022-10-24-npm-v9-0-0-released/#timeline-to-ga) makes this a non-issue anyway.